src/main/java/esa/sen2vm/utils/Sen2VMFilesConstants.java

```diff
@@ -0,0 +1,37 @@
+package esa.sen2vm;
```

Comment:
Why do we have 2 different files ?
It might be clearer in one with section ?

Reply:
Done here: https://github.com/sen2vm/sen2vm-core/commit/6365dea45094dc8eaaa439d211322f625e249e26